inst/htmlwidgets/leaflet.yaml

```diff
@@ -22,3 +22,7 @@ dependencies:
     version: 1.3.1 # match leaflet version
     src: "htmlwidgets/lib/rstudio_leaflet"
     stylesheet: rstudio_leaflet.css
+  - name: leaflet-polylineoffset
+    version: 1.1.1 # match leaflet version
+    src: "htmlwidgets/plugins/leaflet-polylineoffset"
+    stylesheet: leaflet.polylineoffset.js
```

Comment:
It should be `script` instead of `stylesheet`. Otherwise the js-file would be added as css-link

Reply:
Isnt the **src** in `/lib` instead of `/plugin`?